docs/source/openshift/index.adoc

```diff
@@ -0,0 +1,13 @@
+[[interacting-with-openshift]]
+= Interacting with OpenShift
+:icons:
+
+Minishift creates a virtual machine (VM) and provisions a local, single-node OpenShift
```

Comment:
Double checking with the rest of docs.openshift.org, it seems we are doing the same as other topics.